turbinia/client.py

```diff
@@ -787,8 +790,8 @@ def format_request_status(
 
   def format_task_status(
       self, instance, project, region, days=0, task_id=None, request_id=None,
-      user=None, all_fields=False, full_report=False,
-      priority_filter=Priority.HIGH, output_json=False):
+      group_id=None, user=None, all_fields=False, full_report=False,
+      priority_filter=Priority.HIGH, output_json=False, report=[]):
```

Comment:
FYI, mutable objects should not be used as default function arguments as they can cause unexpected side effects.   See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for details.